### ISSUE TITLE
Ordinal Regression with Multiple Output CNN for Age Estimation

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#47]
# Ordinal Regression with Multiple Output CNN for Age Estimation #

- Author: Zhenxing Niu, Mo Zhou, Le Wang, Xinbo Gao, Gang Hua
- Origin: [http://www.cv-foundation.org/openaccess/content_cvpr_2016/papers/Niu_Ordinal_Regression_With_CVPR_2016_paper.pdf](http://www.cv-foundation.org/openaccess/content_cvpr_2016/papers/Niu_Ordinal_Regression_With_CVPR_2016_paper.pdf)
- Related:
  - [https://github.com/luoyetx/OrdinalRegression](https://github.com/luoyetx/OrdinalRegression)
  - [kongsicong: 对参考论文Ordinal Regression with Multiple Output CNN for Age Estimation理解](http://kongsicong.github.io/2016/08/09/%E5%B9%B4%E9%BE%84%E8%AF%86%E5%88%AB/)
  - [https://github.com/kongsicong/Age_recognition_OR](https://github.com/kongsicong/Age_recognition_OR)
  - [AFAD Dataset](http://afad-dataset.github.io/)

> we propose an End-to-End learning approach to address ordinal regression problems using deep Convolutional Neural Network, which could simultaneously conduct feature learning and regression modeling

> we publish an Asian Face Age Dataset (AFAD) containing more than 160K facial images with precise age ground-truths, which is the largest public age dataset to date.

> this is the first work to address ordinal regression problems by using CNN, and achieves the state-of-the-art performance on both the MORPH and AFAD datasets

## dataset ##

- MORPH II: https://ebill.uncw.edu/C20231_ustores/web/store_main.jsp?STOREID=4
- AFAD:
  - http://afad-dataset.github.io/
  - https://github.com/afad-dataset/tarball

## training, testing set ##

> Following the experimental setting in [6][7][31], for both MORPH II and AFAD datasets, we randomly divide the whole dataset into two parts: one part (i.e., 80% of the whole data) is used for training, and the other one (i.e., 20% of the whole data) is used for testing.

## performance ##

> The performance is measured by the Mean Absolute Error (MAE) metric and the Cumulative Score (CS).

**Comment by joyhuang9473** [comment]
The AFAD Dataset: The resource is under preparation.

**Comment by joyhuang9473** [comment]
Openbr: Open Source Biometrics, Face Recognition http://www.openbiometrics.org https://github.com/biometrics/openbr

- openbr/app/examples/age_estimation.cpp: https://github.com/biometrics/openbr/blob/master/app/examples/age_estimation.cpp

**Comment by joyhuang9473** [comment]
angus.ai: Age and Gender Estimation http://angus-doc.readthedocs.io/en/latest/services/agegenderestimation/agegenderestimation.html

- angus-ai/angus-sdk-python: https://github.com/angus-ai/angus-sdk-python

**Comment by joyhuang9473** [comment]
DAGER: Deep Age, Gender and Emotion Recognition Using Convolutional Neural Networks https://github.com/number9473/datanmachlearn/issues/123

**Comment by joyhuang9473** [comment]
2016 Looking at People CVPR Challenge http://gesture.chalearn.org/2016-looking-at-people-cvpr-challenge

Session II: Winners Apparent Age Estimation  (Challenge Track 1) http://gesture.chalearn.org/2016-looking-at-people-cvpr-challenge/workshop

**Comment by DinoHubber** [comment]
Hi! Any updates on when the AFAD dataset resource will be available? Thank you :)

**Comment by John-niu-07** [comment]
AFAD is available for download now http://afad-dataset.github.io/
